### PR TITLE
local instructions, only captains cancel missions, email typo

### PR DESCRIPTION
**Pull request by svetlichnaya**
* full instructions to start running locally
* only captains should see the option to cancel a mission
* fix small email typo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,9 +91,20 @@ $ inv run.gunicorn
 
 The site will be accessible on localhost port 9876 (it's a countdown!).
 
+The full sequence of commands to start running a local version of the site:
+```
+inv run.mysql
```
Comment (igor47) [inline README.md:98]: do you want to add  `workon pyspaceship` and `pip install -r requirements.txt`

**Review by igor47 — APPROVED** [review]
lgtm!

File: README.md (hunk 1 above)
Comment (svetlichnaya) [reply README.md:98]: yes good idea for completeness

**Review by igor47 — APPROVED** [review]
looks even better now!

**Pull request merged**